### ISSUE TITLE
Content & IA review from Claims & Appeals

### ISSUE THREAD

**Issue by Mottie** [department-of-veterans-affairs/va.gov-team#39900]
## What does your team need support for? Check all that apply.
- [ ] Launching one or more new unauthenticated (static) VA.gov pages
- [ ] Making revisions to one or more existing unauthenticated (static) VA.gov page
- [ ] Launching a new online form, app, or tool on VA.gov
- [x] Making revisions to an existing online form, app, or tool on VA.gov
- [ ] Something else (please add details in the next question)

## What do you need help with?
  - [x] Content support
  - [x] IA support
  
The [letters app page](https://staging.va.gov/records/download-va-letters/letters/letter-list) contains two links to eBenefits. One for requesting a Certificate of Eligibility (COE) and another to request a copy of the Veterans DD 214.

We need to replace the COE link to the on VA.gov (/housing-assistance/home-loans/request-coe-form-26-1880 maybe?) and a link to get their DD 214 (unknown). Additionally, the content of those links will need to be reviewed & updated.

<img width="635" alt="Screen Shot 2022-04-12 at 3 05 02 PM" src="https://user-images.githubusercontent.com/136959/163044881-463daf76-ece7-4946-8635-4ee573890346.png">
  
## Supporting Artifacts
*Please provide supporting artifacts as available.*

- Link to product outline: https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/products/benefit-letters
- Link to or attach user flows:
- Link to prototype or mockups:
- Link to any epics/issues that may be helpful:

## Will this work be going through the Collaboration Cycle?
- [ ] Yes 
- [x] No

## When does this work need to be done?
- Estimated launch date:
- Estimated staging review date:
- Content and IA work needed by: No set date

## Do you plan to bring this to an upcoming content office hours session?
- [ ] Yes, benefit content office hours (Thursdays, 3:00 p.m. to 3:45 p.m. ET)
- [ ] Yes, health  content office hours (Thursdays, 11:00 a.m. to 11:30 a.m. ET)
- [ ] Yes, unauth office hours (Mondays, 11:00 a.m. to 11:30 a.m. ET)
- [ ] No, but I'd like to schedule time to talk about this request
- [x] No, let's work asynchronously and meet if needed

**Note:** If we think this work would benefit from a collaborative session with you, we may ask you to bring it to office hours or set up a separate time to meet.

## About your team

- Team name: Claims & Appeals
-	OCTO-DE product owner: Matt Self
-	Product manager: TBD (in transition)
-	Designer: TBD (in transition)
-	FE engineer: @Mottie   
-	Product/team Slack channel: #vsa-claims-appeals

## Next steps

**Once you’ve submitted this ticket, please post a link to this issue in the [#va-sitewide-content-ia](https://dsva.slack.com/channels/va-sitewide-content-ia) Slack channel and tag Beth Potts.**

If you also need engineering support from the Public Websites team, fill out their [intake request form](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=Public+Websites%2C+Sitewide+content&labels=vsa-public-websites%2C+vsa%2C+vsa-public-websites-intake%2C+sitewide-content%2C+needs-grooming&template=public-websites-intake.md&title=%3CType+of+Request%3E+from+%3CTeam%3E).
  
If you need a page/URL redirected, a URL changed or a vanity URL set up, please submit a [Redirect, URL change, or vanity URL request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=mnorthuis&labels=ia&template=redirect-request.md&title=Redirect+Request)

**Comment by mnorthuis** [comment]
For the DD214, we can link them to our content page explaining how to get those records:
https://www.va.gov/records/get-military-service-records/

For the home loan COE, that updated tool on va.gov hasn't launched yet. 

The static page supporting the letters tool should also be reviewed to direct users appropriately.

**Comment by bethpottsVADEPO** [comment]
@Mottie For the DD214, the text can say this:
[Request your military service records (including DD214)](https://www.va.gov/records/get-military-service-records/)

Can we just get rid of the COE link for now--and keep this ticket open (or open a new one) for when we do have a place to link to (when COE launches)?

**Comment by mnorthuis** [comment]
@Mottie @bethpottsVADEPO Is this resolved?

**Comment by Mottie** [comment]
@mnorthuis Sorry, no. We still haven't launched COE

**Comment by RLHecht** [comment]
@Mottie now that COE has launched is this link updated and can this be closed?

**Comment by Mottie** [comment]
Yeah, I think we can close this... this is what I'm seeing now:

- [View and print your Post-9/11 GI Bill statement of benefits.](https://staging.va.gov/education/gi-bill/post-9-11/ch-33-benefit)
- [Request your military service records (including DD214).](https://staging.va.gov/records/get-military-service-records/)

<img width="727" alt="A lot of people come to this page looking for their Post-9/11 GI Bill statement of benefits, their Certificate of Eligibility (COE) for home loan benefits, and their DD214. We don’t have these documents available here yet, but if you’re eligible for them, you can get them through these links: View and print your Post-9/11 GI Bill statement of benefits. Request your military service records (including DD214)." src="https://user-images.githubusercontent.com/136959/225646915-40fbebf8-4033-42eb-8d00-d7b4732344be.png">

**Comment by RLHecht** [comment]
Does there need to be a link to COE now that it's live and we mention it in that paragraph?

**Comment by Mottie** [comment]
Yeah, I guess we need the link content then - I'm not sure which team owns this app now
